### PR TITLE
[hi] Add content/hi/docs/tutorial/kubernetes-basics/_index.html

### PR DESCRIPTION
**Pull request by rajatgupta24**
Hey folks, this is a PR for hi localization of the file: `content/hi/docs/tutorial/kubernetes-basics/_index.html`
PR is linked under this issue #29353

/language hi

### CODE REVIEW

**Review by verma-kunal — COMMENTED** [review]
LGTM 👍

**Comment by anubha-v-ardhan** [comment]
Please note that the directory name is `tutorials` and not `tutorial`.

**Comment by rajatgupta24** [comment]
> Please note that the directory name is `tutorials` and not `tutorial`. Change the name of directory to avoid conflicts in future : ) Thanks

Sorry, I made the changes, should I made the PR ready, I mean to move from draft mode.

**Comment by anubha-v-ardhan** [comment]
@rajatgupta24 No, keep the PR in draft mode for now.
Also, I can't see any files in File changed section.

**Comment by rajatgupta24** [comment]
> @rajatgupta24 No, keep the PR in draft mode for now. We will open PRs for review once we get a separate branch.
> Also, I can't see any files in `Files changed` section.

@anubha-v-ardhan, check now

File: content/hi/docs/tutorials/kubernetes-basics/_index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      </div>
+    </div>
+
+    <br>
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>कुबेरनेट्स आपके लिए क्या कर सकता है?</h2>
+        <p>आधुनिक वेब सेवाओं के साथ, उपयोगकर्ता उम्मीद करते हैं कि एप्लिकेशन 24/7 उपलब्ध होंगे, और डेवलपर्स उन एप्लिकेशन के नए संस्करणों को दिन में कई बार तैनात करने की उम्मीद करते हैं। कंटेनरीकरण पैकेज सॉफ़्टवेयर को इन लक्ष्यों को पूरा करने में मदद करता है, जिससे एप्लिकेशन बिना डाउनटाइम के रिलीज़ और अपडेट किए जा सकते हैं। Kubernetes आपको यह सुनिश्चित करने में मदद करता है कि वे कंटेनरीकृत एप्लिकेशन जहां और जब आप चाहते हैं, चलते हैं, और उन्हें उन संसाधनों और उपकरणों को खोजने में मदद करते हैं जिनकी उन्हें काम करने की आवश्यकता होती है। कुबेरनेट्स एक प्रोडक्शन-रेडी, ओपन सोर्स प्लेटफॉर्म है जिसे कंटेनर ऑर्केस्ट्रेशन में गूगल के संचित अनुभव के साथ डिज़ाइन किया गया है, जो समुदाय के सर्वश्रेष्ठ-नस्लीय विचारों के साथ संयुक्त है।</p>
```
Comment (sftim) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:44]: nit: Would a Hindi reader confidently understand “24/7”? Are there some readers who would and others who might not?

Comment (anubha-v-ardhan) [reply content/hi/docs/tutorials/kubernetes-basics/_index.html:44]: I think everyone will easily understand 24/7. It is possible to write numbers in devnagri but in real life, they are almost extinct and everyone just uses Latin 1, 2, 3 ,4 ...
24/7 is a common term used in India too.

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/tutorial/kubernetes-basics/_index.html

**Comment by anubha-v-ardhan** [comment]
@rajatgupta24 You can change the base branch to `dev-1.22-hi.1`

File: content/hi/docs/tutorials/kubernetes-basics/_index.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+  <main class="content">
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>बुनियादी कुबेरनेट्स</h2>
+        <p>यह ट्यूटोरियल कुबेरनेट्स क्लस्टर ऑर्केस्ट्रेशन सिस्टम की मूल बातों का पूर्वाभ्यास प्रदान करता है। प्रत्येक मॉड्यूल में कुबेरनेट्स की प्रमुख विशेषताओं और अवधारणाओं पर कुछ पृष्ठभूमि की जानकारी होती है, और इसमें एक इंटरैक्टिव ऑनलाइन ट्यूटोरियल शामिल होता है। ये इंटरेक्टिव ट्यूटोरियल आपको अपने लिए एक साधारण क्लस्टर और इसके कंटेनरीकृत अनुप्रयोगों का प्रबंधन करने देते हैं।</p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:27]:
```suggestion
        <p>यह ट्यूटोरियल कुबेरनेट्स क्लस्टर ऑर्केस्ट्रेशन सिस्टम की मूल बातों का पूर्वाभ्यास प्रदान करता है। प्रत्येक मॉड्यूल में कुबेरनेट्स की प्रमुख विशेषताओं और अवधारणाओं पर कुछ जानकारी और एक इंटरैक्टिव ऑनलाइन ट्यूटोरियल शामिल होते हैं। ये इंटरेक्टिव ट्यूटोरियल आपको अपने लिए एक साधारण क्लस्टर और इसके कंटेनरीकृत एप्लीकेशन का प्रबंधन करने देते हैं।</p>
```

File: content/hi/docs/tutorials/kubernetes-basics/_index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        <p>इंटरैक्टिव ट्यूटोरियल का उपयोग करके, आप सीख सकते हैं:</p>
+        <ul>
+          <li>एक क्लस्टर पर एक कंटेनरीकृत अनुप्रयोग डिप्लॉय करें।</li>
+          <li>डिप्लॉयमेंट को स्केल करें।</li>
+          <li>एक नए सॉफ़्टवेयर संस्करण के साथ कंटेनरीकृत एप्लिकेशन को अपडेट करें।</li>
+          <li>कंटेनरीकृत एप्लिकेशन को डीबग करें।</li>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:33]:
```suggestion
          <li>एक क्लस्टर पर कंटेनरीकृत एप्लीकेशन डिप्लॉय करें।</li>
          <li>डिप्लॉयमेंट को स्केल करें।</li>
          <li>नए सॉफ़्टवेयर संस्करण के साथ कंटेनरीकृत एप्लिकेशन को अपडेट करें।</li>
          <li>कंटेनरीकृत एप्लिकेशन को डीबग करें।</li>
```

File: content/hi/docs/tutorials/kubernetes-basics/_index.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        </ul>
+        <p>ट्यूटोरियल आपके वेब ब्राउज़र में एक वर्चुअल टर्मिनल चलाने के लिए कटाकोडा का उपयोग करते हैं जो मिनिक्यूब चलाता है, कुबेरनेट्स की एक छोटे पैमाने पर स्थानीय डिप्लॉयमेंट जो कहीं भी चल सकती है। कोई सॉफ़्टवेयर स्थापित करने या कुछ भी कॉन्फ़िगर करने की कोई आवश्यकता नहीं है; प्रत्येक इंटरैक्टिव ट्यूटोरियल सीधे आपके वेब ब्राउज़र से ही चलता है।</p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:35]:
```suggestion
        <p>ट्यूटोरियल आपके वेब ब्राउज़र में एक वर्चुअल टर्मिनल चलाने के लिए कटाकोडा का उपयोग करते हैं जो मिनिक्यूब नामक कुबेरनेट्स का एक छोटा स्थानीय डिप्लॉयमेंट चलाता है। कोई सॉफ़्टवेयर स्थापित करने या कुछ भी कॉन्फ़िगर करने की कोई आवश्यकता नहीं है; प्रत्येक इंटरैक्टिव ट्यूटोरियल सीधे आपके वेब ब्राउज़र से ही चलता है।</p>
```

File: content/hi/docs/tutorials/kubernetes-basics/_index.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+      </div>
+    </div>
+
+    <br>
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>कुबेरनेट्स आपके लिए क्या कर सकता है?</h2>
+        <p>आधुनिक वेब सेवाओं के साथ, उपयोगकर्ता उम्मीद करते हैं कि एप्लिकेशन 24/7 उपलब्ध होंगे, और डेवलपर्स उन एप्लिकेशन के नए संस्करणों को दिन में कई बार डिप्लॉय करने की उम्मीद करते हैं। कंटेनरीकरण पैकेज सॉफ़्टवेयर को इन लक्ष्यों को पूरा करने में मदद करता है, जिससे एप्लिकेशन बिना डाउनटाइम के रिलीज़ और अपडेट किए जा सकते हैं। कुबेरनेट्स आपको यह सुनिश्चित करने में मदद करता है कि वे कंटेनरीकृत एप्लिकेशन जहां और जब आप चाहते हैं, चलते हैं, और उन्हें उन संसाधनों और उपकरणों को खोजने में मदद करते हैं जिनकी उन्हें काम करने की आवश्यकता होती है। कुबेरनेट्स एक प्रोडक्शन-रेडी, ओपन सोर्स प्लेटफॉर्म है जिसे कंटेनर ऑर्केस्ट्रेशन में गूगल के संचित अनुभव के साथ डिज़ाइन किया गया है, जो समुदाय के सर्वश्रेष्ठ-नस्लीय विचारों के साथ संयुक्त है।</p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:44]:
```suggestion
        <p>आधुनिक वेब सेवाओं के साथ उपयोगकर्ता उम्मीद करते हैं कि एप्लिकेशन 24/7 उपलब्ध होंगे, और डेवलपर्स उन एप्लिकेशन के नए संस्करणों को दिन में कई बार डिप्लॉय करने की उम्मीद करते हैं। कंटेनरीकरण पैकेज सॉफ़्टवेयर को इन लक्ष्यों को पूरा करने में मदद करता है, जिससे एप्लिकेशन बिना डाउनटाइम के रिलीज़ और अपडेट किए जा सकते हैं। कुबेरनेट्स आपको यह सुनिश्चित करने में मदद करता है कि वे कंटेनरीकृत एप्लिकेशन जहां और जब आप चाहते हैं, चलते हैं, और उन्हें उन संसाधनों और उपकरणों को खोजने में मदद करते हैं जिनकी उन्हें काम करने के लिए आवश्यकता होती है। कुबेरनेट्स एक उत्पादन के लिए तैयार, ओपन सोर्स प्लेटफॉर्म है जिसे कंटेनर ऑर्केस्ट्रेशन में गूगल के संचित अनुभव के साथ डिज़ाइन किया गया है और समुदाय के उत्तम विचारों से संयुक्त है।</p>
```

File: content/hi/docs/tutorials/kubernetes-basics/_index.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. अपना ऐप बढ़ाएं</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/update/update-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_06.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/update/update-intro/"><h5>6. अपना ऐप अपडेट करें</h5></a>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:103]:
```suggestion
                <a href="/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><h5>1. कुबेरनेट्स क्लस्टर बनाएं</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_02.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><h5>2. ऐप डिप्लॉय करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/explore/explore-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_03.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/explore/explore-intro/"><h5>3. अपना ऐप एक्सप्लोर करें</h5></a>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div class="col-md-12">
          <div class="row">
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. अपने ऐप को सार्वजनिक रूप से एक्सपोज़ करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_05.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. अपना ऐप स्केल करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/update/update-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_06.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/update/update-intro/"><h5>6. अपना ऐप अपडेट करें</h5></a>
```

Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:103]:
```suggestion
                <a href="/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><h5>1. कुबेरनेट्स क्लस्टर बनाएं</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_02.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><h5>2. ऐप डिप्लॉय करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/explore/explore-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_03.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/explore/explore-intro/"><h5>3. अपना ऐप एक्सप्लोर करें</h5></a>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div class="col-md-12">
          <div class="row">
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. अपने ऐप को सार्वजनिक रूप से एक्सपोज़ करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_05.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. अपना ऐप स्केल करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/update/update-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_06.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/update/update-intro/"><h5>6. अपना ऐप अपडेट करें</h5></a>
```

Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:103]: Can you change all the links to `hi` ones, as these pages are part of our initial localization
For example:
`/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/` -> `/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/`

File: content/hi/docs/tutorials/kubernetes-basics/_index.html [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. अपने ऐप को सार्वजनिक रूप से एक्सपोज़ करें</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_05.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. अपना ऐप बढ़ाएं</h5></a>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/_index.html:95]:
```suggestion
                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. अपना ऐप स्केल करें</h5></a>
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
You can change the base to `dev-1.22-hi.1`

File: content/hi/docs/tutorials/kubernetes-basics/_index.html (hunk 6 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/tutorials/kubernetes-basics/_index.html:103]: We can link to Hindi version as these pages are part of our initial localization

**Pull request merged**